### ISSUE TITLE
Bring v0.3.0 to Homebrew

### ISSUE THREAD

**Issue by siddharthist** [pote/gvp#38]
Homebrew needs some love with a new version. @pote hinted that they would head this effort, since I don't run or have access to OSX. Other testers would be appreciated.

**Comment by nikolay** [comment]
@pote :pray:

**Comment by pote** [comment]
Sorry folks, work has been insane, I now possess a bit of the coveted thing people call "free time", so I'll work on this today. :smile: :dancer:

**Comment by pote** [comment]
I'm trying to get homebrew's gvp updated, but currently the install process seems to be broken on OSX, which is weird, can anybody confirm if they can get `v0.3.0` (or any newer version) installed and running?

**Comment by djui** [comment]
GVP is still on `0.2.0`:

```
$ brew info gvp
gvp: stable 0.2.0
Go versioning packager
https://github.com/pote/gvp
/usr/local/Cellar/gvp/0.2.0 (4 files, 16K) *
  Built from source
From: https://github.com/Homebrew/homebrew/blob/master/Library/Formula/gvp.rb
```

**Comment by djui** [comment]
In fact, trying to run 0.3.0 shows syntax error:

$ gvp init
/usr/local/bin/gvp: line 106: syntax error near unexpected token `;'
/usr/local/bin/gvp: line 106: `        ;&'

`source gvp` works, but every consecutive calls just close the shell:

```
$ source gvp
>> Local GOPATH set.

(gap)$ gvp version
>> gvp v0.3.0
[TERMINATED]
```

**Comment by siddharthist** [comment]
@djui You may have an old version of Bash, there is syntax in that line that is introduced in version 4.0.

**Comment by djui** [comment]
I can check later, but it's El Capitan out of the box, which I hope Apple ships with 4+.

**Comment by djui** [comment]
Besides, is there any reason to rely on that feature?

**Comment by siddharthist** [comment]
@djui Not necessarily, PR #46 will eliminate that version dependency when it's complete/merged. Thanks for reporting this, I didn't realize that OSX shipped with Bash < 4.0.
